### PR TITLE
Add support for tinyint. Fix bool issue.

### PR DESCRIPTION
**Pull request by dereuromark**
Part of https://github.com/cakephp/migrations/pull/457

Needed for https://github.com/cakephp/migrations/issues/355

### CODE REVIEW

**Comment by dereuromark** [comment]
I pushed some CS fixes in a separate branch to easier verify and review these here
https://github.com/cakephp/phinx/compare/tinyint-cs

File: src/Phinx/Db/Adapter/MysqlAdapter.php [hunk 1]
```diff
@@ -956,6 +957,8 @@ public function getSqlType($type, $limit = null)
                 return ['name' => 'bit', 'limit' => $limit ?: 64];
             case static::PHINX_TYPE_SMALL_INTEGER:
                 return ['name' => 'smallint', 'limit' => $limit ?: 6];
+            case static::PHINX_TYPE_TINY_INTEGER:
+                return ['name' => 'tinyint', 'limit' => $limit ?: 3];
```
Comment (ADmad) [inline src/Phinx/Db/Adapter/MysqlAdapter.php:961]: Isn't the default limit/length for tinyint in MySQL "4"?

Comment (dereuromark) [reply src/Phinx/Db/Adapter/MysqlAdapter.php:961]: Is it? I thought 0...255 would be 3, but since it will fill one byte anyway you might be right.

Comment (dereuromark) [reply src/Phinx/Db/Adapter/MysqlAdapter.php:961]: I did some research and the length here is the max display width, so given the negative char `-128` it seems to be 4 then.

**Comment by dereuromark** [comment]
Whats wrong with stickler these days?

**Comment by dereuromark** [comment]
Ah, now it is green. Maybe some PR change is necessary to update the push result.

**Pull request merged**